gerrit_to_github_issues/github_issues.py

```diff
@@ -75,3 +76,52 @@ def get_bot_comment(issue: Issue, bot_name: str, ps_number: str) -> IssueComment
     for i in issue.get_comments():
         if i.user.login == bot_name and ps_number in i.body:
             return i
+
+
+def assign_issues(repo: github.Repository):
+    open_issues = [i for i in repo.get_issues() if i.state == 'open']
+    for issue in open_issues:
+        try_assign(issue)
+
+
+def try_assign(issue: github.Issue):
+    # find the most recent assignment request
+    assignment_request = None
+    for comment in issue.get_comments().reversed:
+        if '/assign' in comment.body:
+            assignment_request = comment
+            break
+    if not assignment_request:
+        # Looks like no one wants this issue
+        return
+
+    if not issue.assignees:
+        # If no one has been assigned yet, let the user take the issue
+        issue.add_to_assignees(assignment_request.user)
+        issue.create_comment(f'assigned {assignment_request.user.login}')
+        return
+
+    if issue_age(issue) > 30:
+        # If the issue is 2 months old and the original assignees haven't
```

Comment:
age is only being compared to 1 month but the comment suggests 2 months.

Reply:
fixed